.github/workflows/deploy_to_dev.yml

```diff
@@ -18,6 +18,12 @@ jobs:
       - name: set-up-environment
         uses: DFE-Digital/github-actions/set-up-environment@master
 
+      - name: Set up Node
```

Comment:
I was reasoning that the workflows are very similar so it would probably be beneficial in both; happy to pull this out though and we can look at it if/when it arises on the dev deploy 👍